Lib/lib2to3/refactor.py

```diff
@@ -107,22 +107,6 @@ def get_fixers_from_package(pkg_name):
 def _identity(obj):
     return obj
 
-if sys.version_info < (3, 0):
```

Comment:
io.open() is available since Python 2.6, so my change should work on Python 2.6 and newer.

I don't know why this file has support for Python 2.